mus: Add missing dependencies for internal gpu interfaces.

GpuMain depends on display_compositor in cc, and GpuService depends
on gfx and geometry mojoms.

BUG=643746

Review-Url: https://codereview.chromium.org/2583933002
Cr-Commit-Position: refs/heads/master@{#439162}
diff --git a/services/ui/gpu/interfaces/BUILD.gn b/services/ui/gpu/interfaces/BUILD.gn
index 3fbc22e4..c1c6503 100644
--- a/services/ui/gpu/interfaces/BUILD.gn
+++ b/services/ui/gpu/interfaces/BUILD.gn
@@ -13,8 +13,11 @@
   ]
 
   public_deps = [
+    "//cc/ipc:internal_interfaces",
     "//gpu/ipc/common:interfaces",
     "//services/ui/public/interfaces",
+    "//ui/gfx/geometry/mojo",
+    "//ui/gfx/mojo",
     "//url/mojo:url_mojom_gurl",
   ]
 }

